schema.json

```
@@ -1005,87 +1059,113 @@
             "type": "string"
         },
         "entity": {
-            "type": "object",
+            "description": "An entity",
```

```suggestion
            "description": "An entity, i.e., an institution, team, research group, company, conference, etc., as opposed to a single natural person.",
```

Extend description to make clear what "entity" means here.